localization/english/wc_decisions_l_english.yml

```diff
@@ -13,3 +13,16 @@
  
  create_gorian_empire_de_jure_changes:1 "Becomes [de_jure|E] [kingdoms|E] of the new Gorian Empire"
  create_gorian_empire:1 "You #P restore#! #V The Gorian Empire#!"
+
+ borrow_loan_goblin_decision:0 "Request Loan from the Goblin Bank"
+ borrow_loan_goblin_decision_tooltip:0 "Request a Loan through the nearest Goblin Bank branch"
+ borrow_loan_goblin_decision_desc:0 "One can only hope that this won't be too complicated of a process, but the Goblins aren't exactly known for making matters too easy for folks who want them to throw money out of their pockets. But, the machinations that have been carefully planned for the future of these lands certainly deserve some external investement. The future... it can only belong to those who are strong enough to stride boldly towards it, and if this isn't a bold stride, what is? The Goblins will likely see this as a new age of some sort of a dubious partnership, and hope that when comes time, we are forced to remember their gestures of goodwill."
```

Comment:
I'd go here with "and hope that when the time comes", but I think it's ok like this, too.